test/terra/reference/ref_measure.py

```diff
@@ -309,3 +308,33 @@ def measure_counts_qobj_nondeterministic(shots, hex_counts=True):
         targets.append({'000': shots / 4, '001': shots / 4,
                         '010': shots / 4, '011': shots / 4})
     return targets
+
+
+# ==========================================================================
+# Conditional measure
+# ==========================================================================
+
+def measure_conditional():
+    """"Measure test circuits with conditions."""
+
+    circuits = []
+    qr = QuantumRegister(1)
+    cr = ClassicalRegister(1)
+
+    # Measure |++> state (sampled)
+    circuit = QuantumCircuit(qr, cr)
+    circuit.x(qr[0])
+    circuit.barrier(qr)
+    circuit.measure(qr, cr).c_if(c, 1)
```

Comment:
Seems like `c` variable in: `.c_if(c, 1)` is not defined, I guess that you meant: `cr` ?